# Don't unconditionally strip the binary, let dh_strip do the work correctly.

--- Makefile.orig	2008-01-24 13:32:46.000000000 +0100
+++ Makefile	2008-01-24 13:35:29.000000000 +0100
@@ -36,7 +36,6 @@
 
 $(PROGRAM): $(OBJS)
 	$(CC) -o $(PROGRAM) $(OBJS) $(LDFLAGS)
-	$(STRIP) $(STRIP_ARGS) $(PROGRAM)
 
 clean:
 	rm -f *.o *~
